filestore/api.py

```diff
@@ -10,8 +10,8 @@
 
 logger = logging.getLogger(__name__)
 
-
-_FS_SINGLETON = FileStore(connection_config)
+VERSION = 1
+_FS_SINGLETON = FileStore(connection_config, version=VERSION)
```

Comment:
Following MDS, make sure that the singleton in `filestore.api` does not bump to version until the databases are migrated.